api/app/tests/apps/signals/test_managers.py

```diff
@@ -45,3 +50,138 @@ def test_update_multiple_invalid_category_data(self):
             )
 
         self.assertEqual(ve.exception.message, 'Category not found in data')
+
+    def test_create_initial_default_type(self):
+        signal_data = {
+            'text': 'Bladiebla',
+            'text_extra': 'Meer bladiebla',
+            'incident_date_start': '2020-02-26T12:00:00.000000Z',
+            'source': 'online',
+        }
+        location_data = {
+            'geometrie': Point(4.90022563, 52.36768424)
+        }
+        category_assignment_data = {
+            'category': self.category
+        }
+        status_data = {}  # Default status
+        reporter_data = {}  # No reporter
+        priority_data = None  # Default priority
+        type_data = None  # This triggers the default Type
+
+        signal = Signal.actions.create_initial(signal_data=signal_data,
+                                               location_data=location_data,
+                                               status_data=status_data,
+                                               category_assignment_data=category_assignment_data,
+                                               reporter_data=reporter_data,
+                                               priority_data=priority_data,
+                                               type_data=type_data)
+
+        self.assertEqual(signal.types.count(), 1)
+        self.assertIsNotNone(signal.type_assignment)
+        self.assertEqual(signal.type_assignment.name, Type.SIGNAL)  # Default is SIGNAL
```

Comment:
Maybe add this as a constant in the tests, so that it is not a magic value (here and below).